lib/dl_core/dl_core/us_manager/us_manager_async.py

```diff
@@ -154,6 +154,9 @@ async def save(self, entry: USEntry) -> None:
             entry._stored_in_db = True
         else:
             # noinspection PyProtectedMember
+            if update_revision is not None:
```

Comment:
you can just add it anyway, if `update_revision is None` it won't be added to `json_data`